docs/en/query_language/agg_functions/reference.md

```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
 
 Don't use this function for calculating timings. There is a more suitable function for this purpose: `quantileTiming`.
 
-## quantileTiming(level)(x)
+## quantileTiming {#agg_function-quantiletiming}
 
-Computes the quantile of 'level' with a fixed precision.
-Works for numbers. Intended for calculating quantiles of page loading time in milliseconds.
+Computes the quantile of the specified level with determined precision. The function intended for calculating quantiles of page loading time in milliseconds. 
 
-If the value is greater than 30,000 (a page loading time of more than 30 seconds), the result is equated to 30,000.
+```
+quantileTiming(level)(expr)
+```
+
+**Accuracy**
+
+The calculation is accurate if:
+
+- Total number of values is not more than about 5670.
+- Total number of values is more than about 5670, but the times of page loading is less than 1024ms.
+
+Otherwise, the result of a calculation is rounded to the value, multiple of 16 ms.
+
+For calculating quantiles of page loading times, using this function is more effective, and the result is more accurate than for the [quantile](#agg_function-quantile) function.
+
+**Parameters**
 
-If the total value is not more than about 5670, then the calculation is accurate.
+- `level` — Quantile level. Range: [0, 1].
+- `expr` — [Expression](../syntax.md#syntax-expressions) returning number in the [Float*](../../data_types/float.md) type.
```

Comment:
we'd better explicitly mention that it's supposed to be in milliseconds (btw why it's float then?)

Reply:
From the dev pull: "Returned back support for floating point argument in function quantileTiming for backward compatibility."